### PR TITLE
Set Table Cell props default value to `null`

### PR DESCRIPTION
**Pull request by zxy-c**

### CODE REVIEW

File: muirwik-components/src/main/kotlin/com/ccfraser/muirwik/components/table/TableCell.kt [hunk 1]
```diff
@@ -67,26 +70,34 @@ var MTableCellProps.variant by EnumPropToString(MTableCellVariant.values())
 
 fun RBuilder.mTableCell(
         key: Any? = null,
-        variant: MTableCellVariant = MTableCellVariant.body,
-        sortDirection: MTableCellSortDirection = MTableCellSortDirection.False,
-        align: MTableCellAlign = MTableCellAlign.inherit,
-        padding: MTableCellPadding = MTableCellPadding.default,
+        variant: MTableCellVariant? = null,
+        sortDirection: MTableCellSortDirection? = null,
+        align: MTableCellAlign? = null,
```
Comment (cfnz) [inline muirwik-components/src/main/kotlin/com/ccfraser/muirwik/components/table/TableCell.kt:75]: inherit is the default in Material UI docs, so this one probably should stay.

File: muirwik-components/src/main/kotlin/com/ccfraser/muirwik/components/table/TableCell.kt [hunk 2]
```diff
@@ -54,11 +54,14 @@ interface MTableCellProps : StyledPropsWithCommonAttributes {
     @JsName("sortDirection")
     var rawSortDirection: dynamic
 }
+
 var MTableCellProps.align by EnumPropToString(MTableCellAlign.values())
 var MTableCellProps.padding by EnumPropToString(MTableCellPadding.values())
 var MTableCellProps.size by EnumPropToString(MTableCellSize.values())
 var MTableCellProps.sortDirection: MTableCellSortDirection
-    get() = if (rawSortDirection == false) MTableCellSortDirection.False else MTableCellSortDirection.valueOf(rawSortDirection)
+    get() = if (rawSortDirection == false) MTableCellSortDirection.False else MTableCellSortDirection.valueOf(
```
Comment (cfnz) [inline muirwik-components/src/main/kotlin/com/ccfraser/muirwik/components/table/TableCell.kt:62]: Looks like just line wrapping, so maybe this should stay as it was

File: muirwik-components/src/main/kotlin/com/ccfraser/muirwik/components/table/TableCell.kt [hunk 3]
```diff
@@ -67,26 +70,34 @@ var MTableCellProps.variant by EnumPropToString(MTableCellVariant.values())
...
+        padding: MTableCellPadding? = null,
         size: MTableCellSize? = null,
         colSpan: Int? = null,
         component: String? = null,
         scope: String? = null,
-
         className: String? = null,
-        handler: StyledHandler<MTableCellProps>? = null) = createStyled(TableCellComponent) {
-    attrs.align = align
+        handler: StyledHandler<MTableCellProps>? = null
```
Comment (cfnz) [inline muirwik-components/src/main/kotlin/com/ccfraser/muirwik/components/table/TableCell.kt:82]: Line wrapping should probably stay as it was (consistent with the rest of the library)

File: muirwik-components/src/main/kotlin/com/ccfraser/muirwik/components/table/TableCell.kt [hunk 4]
```diff
@@ -67,26 +70,34 @@ var MTableCellProps.variant by EnumPropToString(MTableCellVariant.values())
...
+) = createStyled(TableCellComponent) {
+    if (align != null) {
```
Comment (cfnz) [inline muirwik-components/src/main/kotlin/com/ccfraser/muirwik/components/table/TableCell.kt:84]: For these if null params, probably should keep with the rest of the style and be
`align?.let { attrs.align = it }`

**Review by cfnz — CHANGES_REQUESTED** [review]
Thanks for your input... a couple of minor suggestions which you or I could make...

File: muirwik-components/src/main/kotlin/com/ccfraser/muirwik/components/table/TableCell.kt (hunk 4 above)
Comment (zxy-c) [reply muirwik-components/src/main/kotlin/com/ccfraser/muirwik/components/table/TableCell.kt:84]: em..Should keep same style, I just used quick fix with idea

**Pull request closed without merging**